src/components/APIkey.js

```diff
@@ -0,0 +1,2 @@
+const apiKey = "eqaKx38o0Fg1f/rZqOA/sg==epV9zWMwWAegkqnF";
+export default apiKey;
```

Comment:
- [ ] Storing API keys or other sensitive information on a git repository should be avoided at all costs. Please make use of environment variables in a `.env` file.
Please use this [resource](https://dev.to/felixdusengimana/environment-variables-in-reactjs-139l) if you need some help 😎

Reply:
@codepantha thank you for your review.
but I am unable to do it. Please guide me. I have gone through the link  you provided but unable to import it.

Reply:
@DilsherB I'm sorry for the difficulties you are experiencing. Please reach me on slack or we could have a call on zoom 📹

Reply:
Hi @codepantha 
I couldn't find you on slack. please reach me so that we can go through this.

Reply:
Okay. I just sent you a message

Reply:
Hi @codepantha 
I called you on zoom and waited there for 43 minutes but you didn't appear. I can understand you may be busy but please come up and let it finish.